abdiff/core/run_ab_transforms.py

```diff
@@ -1 +1,187 @@
 """abdiff.core.run_ab_transforms"""
+
+import logging
+import os
+import re
+import time
+from datetime import timedelta
+from pathlib import Path
+from time import perf_counter
+
+import docker
+from docker.models.containers import Container
+
+from abdiff.config import Config
+from abdiff.core.exceptions import DockerContainerRuntimeExceededTimeoutError
+from abdiff.core.utils import create_directories, update_or_create_run_json
+
+CONFIG = Config()
+
+logger = logging.getLogger(__name__)
+
+
+def run_ab_transforms(
+    run_directory: str,
+    image_name_a: str,
+    image_name_b: str,
+    input_files: list[str],
+    docker_client: docker.client.DockerClient | None = None,
+    timeout: int = 180,
```

Comment:
The value for `timeout` gets used by the [`wait_for_containers` function](https://github.com/MITLibraries/transmogrifier-ab-diff/pull/6/files#diff-7a49d9dbf619fca67eb5ed5ed16b6c62440a4cc6898754cd835d2b819be74ddcR133-R136).